### DISCUSSION TITLE
I code a theme for docusaurus(我制作了一个docusaurus主题)

### DISCUSSION THREAD

**Discussion by pincman** [facebook/docusaurus#7717] [Ideas]
source [https://github.com/rowfishjs/rowfish](https://github.com/rowfishjs/rowfish)

**Comment by Josh-Cena** [comment]
This is amazing! Want to add this to our showcase + our resources list?

**Reply by pincman** [reply]
Yes, I have submitted a PR and removed the extra .history folder, thanks

**Reply by pincman** [reply]
@Josh-Cena Could you please help me merge the PR and put my small website in the showcase? thank you very much！

**Reply by Josh-Cena** [reply]
Hey, I was just back from a trip to Guangzhou and I wasn't around for the past few weeks. I'm also considering making this more highlighted—since you have a custom theme, we can put it in the [resources list](https://docusaurus.io/community/resources), or we can give you a new showcase tag (which we've been planning to do anyway). That's why I'm keeping it open for now. Don't worry—I like it very much and we will merge it eventually!

**Reply by pincman** [reply]
thanks, i got it, after the v2 is released, I should refactor this theme and release it as a separate NPM package to ensure faster installation

**Comment by slorber** [comment]
Looks good 👍 except I don't speak Chinese 😅

**Reply by Josh-Cena** [reply]
I have to say the site's (https://pincman.com/) design is very Chinese-style

**Reply by pincman** [reply]
@slorber This is a simple personal website theme, only used to build Chinese sites, publish some technical wikis and video tutorials, personal profiles, etc. For me, it is used to sell some TS/React/Node (Nestjs) related video tutorials Or documentation 😃, this might be an income for me and try to help others can do this too
